pajbot/modules/default_chat_states.py

```diff
@@ -117,6 +135,10 @@ def on_stream_stop(self, **rest) -> bool:
             log.warning("on_stream_stop failed in DefaultChatStatesModule because bot is None")
             return True
 
+        channel_id = self.bot.streamer.id
+        bot_id = self.bot.bot_user.id
+        authorization = self.bot.bot_token_manager
+
```

Comment:
Same thing here

Reply:
The main reason I made these into variables, is that we will need to use these in every single other function in the module. Otherwise we'll be repeating the exact same self.bot... variables in every function.